components/cam/src/chemistry/mozart/mo_drydep.F90

```diff
@@ -2007,7 +2007,6 @@ subroutine get_landuse_and_soilw_from_file(do_soilw)
     
     if (aqua_planet) then
       fraction_landuse = 0.
-      soilw_3d = 0.
```

Comment:
With aqua-planet ON, `soilw_3d` is not allocated, removing: it's allocated with `do_soilw=.true.`. `e3sm_developer` tests still pass after this change.